templates/part.content.php

```diff
@@ -130,7 +130,7 @@ class="icon-toggle keep-unread"
                     </span>
                 </div>
 
-                <div class="enclosure" ng-if="item.enclosureLink">
+                <div class="enclosure" ng-if="Content.getMediaType(item.enclosureMime) == 'audio' || Content.getMediaType(item.enclosureMime) == 'video'">
```

Comment:
Guess it would make sense to duplicate `.enclosure` here to have a dedicated element for audio and video each. Like so we wouldn't need multiple media type checks inside that wrapper

Reply:
Sure, but it's not the point of the issue, anyway i can add it if the team is ok with that ;)

Reply:
I think it's a good idea

Reply:
@SMillerDev shoud i create a separate PR for that or include this in here ?

Reply:
Just do it here, it's not that big a change

Reply:
@SMillerDev & @danopz i've tried it with latest commit, tell me if it's ok